common/PacketTypes.asn1

```diff
@@ -0,0 +1,29 @@
+PacketTypes DEFINITIONS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    PUSC-UINT, PUSC-BOOL FROM BasicTypes
+    ApplicationProcess-ID FROM ApplicationProcess;
+
+
+ApplicationProcess-ID-Packet ::= ApplicationProcess-ID
```

Comment:
why double definition? I'd define proper size and integer range in ApplicationProcess.asn1

Reply:
ApplicationProcess-ID size isn't defined in every message. Should we assume it's always the same size of 11 bits as in packet header?

Reply:
define it in single place plz

you can use "Generics" in PUS1 to allow user for quick substitution of own type